### PR TITLE
PIPRES-208: Payment fee append to subscription recurred order

### PR DESCRIPTION
**Pull request by mandan2**
Fixed issue where payment fee was not appended to recurring orders at all. Logic: fetching payment fee from original order, appending payment fee to duplicated order totals, inserting new payment fee record for the new order.

### CODE REVIEW

File: src/Exception/MollieException.php [hunk 1]
```diff
@@ -12,9 +12,27 @@
...
     const CUSTOMER_EXCEPTION = 1;
 
     const API_CONNECTION_EXCEPTION = 2;
+
+    final public function __construct($message = '', $code = 0, Throwable $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
+
+    public static function unknownError(Exception $exception): self
```
Comment (tomjas1997) [inline src/Exception/MollieException.php:30]: Throwable

File: src/Handler/Order/OrderPaymentFeeHandler.php [hunk 2]
```diff
@@ -37,83 +37,121 @@
...
+                true,
+                Cart::BOTH
+            );
+
+            $originalAmountWithoutTax = (float) $cart->getOrderTotal(
+                false,
+                Cart::BOTH
+            );
+        } catch (Exception $exception) {
+            throw CouldNotHandleOrderPaymentFee::unknownError($exception);
```
Comment (tomjas1997) [inline src/Handler/Order/OrderPaymentFeeHandler.php:117]: Questionable exception, never seen this throw an error

**Review by tomjas1997 — APPROVED** [review]
Seems OK

File: src/Handler/Order/OrderPaymentFeeHandler.php (hunk 2 above)
Comment (mandan2) [reply src/Handler/Order/OrderPaymentFeeHandler.php:117]: Left it anyways, doesn't do any harm atm.

**Pull request merged**